website/src/pages/Blog/index.tsx

```diff
@@ -160,8 +161,8 @@ const Blog: React.FC = () => {
     />
     );
   };
-
-  const getTagsMarkup = Object.keys(tagsDistribution).map((item: string) => {
+  const sortedTags = getTagsSorted(tagsDistribution,"length");
```

Comment:
This sorting also needs to be by name

Reply:
<img width="1440" alt="Screenshot 2021-07-15 at 5 03 21 PM" src="https://user-images.githubusercontent.com/34312966/125781445-18e4ebbb-2154-4b93-aefa-aaa9e3bce4ab.png">